bika/lims/tests/doctests/Permissions.rst

```diff
@@ -278,7 +278,7 @@ Exactly these roles have should have the `List folder contents` permission::
 Exactly these roles have should have the `Modify portal content` permission::
 
     >>> get_roles_for_permission("Modify portal content", labcontacts)
-    ['LabClerk', 'LabManager', 'Manager']
+    ['LabManager', 'Manager']
 
     >>> get_roles_for_permission("Modify portal content", labcontact)
     ['LabClerk', 'LabManager', 'Manager']
```

Comment:
@xispa I am curious about `get_roles_for_permission("Modify portal content", labcontact)`. Should lab clerk have edit permissions for lab contacts?

Reply:
Yes, LabClerk can add and modify Lab contacts. Note this refers to "LabContacts" (the folder), not to the "LabContact" content type. Basically, the test was wrong. Nobody should be able to modify the "LabContacts" **folder** except Manager.